ccmbioinfo/lfs
# CARDS-1893: Publish the Alpine Linux APK package status files for each CARDS Docker image that is in deployment

This PR implements CARDS-1893 by adding the `build_package_listing_docker_image.sh` script which copies, to a new (smaller) image, only the files that are necessary for performing a security vulnerability scan of the Alpine Linux packages that are installed on the (much larger) input Docker image.

To test:

1. Enter the `Utilities/SecurityScanning` directory and run the command `./build_package_listing_docker_image.sh ghcr.io/integralprogrammer/cards:latest cards-apk-listing`
2. Save the built Docker image to a file using `docker save cards-apk-listing -o ~/cards-apk-listing.tar`.
3. Observe that the `~/cards-apk-listing.tar` file is _much_ smaller than the `ghcr.io/integralprogrammer/cards:latest cards-apk-listing` Docker image.
4. Scan the built Docker image with Trivy by running `docker run --rm -v ~/cards-apk-listing.tar:/docker-test.tar aquasec/trivy image --security-checks vuln --ignore-unfixed --input /docker-test.tar` and verify that Trivy is detecting vulnerabilities.
